paywall/next.config.js

```diff
@@ -18,7 +18,6 @@ const copyFile = promisify(fs.copyFile)
 const requiredConfigVariables = {
   unlockEnv: process.env.UNLOCK_ENV || 'dev',
   readOnlyProvider: process.env.READ_ONLY_PROVIDER,
-  paywallUrl: process.env.PAYWALL_URL,
```

Comment:
Can you make sure to removed these values from the Circle config once all this has been merged and deployed please?

Reply:
it will be my pleasure

Reply:
done